modular_skyrat/modules/colony_fabricator/code/design_datums/construction.dm

```diff
@@ -0,0 +1,106 @@
+// Look, I had to make its name start with A so it'd be top of the list, fight me
+
+#define FABRICATOR_SUBCATEGORY_STRUCTURES "/Autofab Structures"
+
+// Techweb node that shouldnt show up anywhere ever specifically for the fabricator to work with
+
+/datum/techweb_node/colony_fabricator_structures
+	id = "colony_fabricator_structures"
+	display_name = "Colony Fabricator Structure Designs"
+	description = "Contains all of the colony fabricator's structure designs."
+	design_ids = list(
+		"prefab_airlock_kit",
+		"prefab_shutters_kit",
+		"prefab_floor_tile",
+		"prefab_cat_floor_tile",
+		"colony_fab_plastic_wall_panel",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000000000000000) // God save you
```

Comment:
All of the techweb nodes for this lathe are a gazillion points because you don't actually research them at all, it's just me being silly because this'll never appear in game